nengo/tests/test_synapses.py

```diff
@@ -54,29 +68,82 @@ def test_lowpass(Simulator, plt, seed, allclose):
     dt = 1e-3
     tau = 0.03
 
-    t, x, yhat = run_synapse(Simulator, seed, Lowpass(tau), dt=dt)
-    y = Lowpass(tau).filt(x, dt=dt, y0=0)
+    synapse = Lowpass(tau)
+    t, x, yhat = run_synapse(Simulator, seed, synapse, dt=dt)
+    y = Lowpass(tau).filt(x, dt=dt)
 
     assert signals_allclose(t, y, yhat, delay=dt, plt=plt, allclose=allclose)
 
+    gains = np.abs(synapse.evaluate([0, synapse.cutoff, 1e16]))
+    assert allclose(gains, [1, 1 / np.sqrt(2), 0])
+
 
 def test_alpha(Simulator, plt, seed, allclose):
     dt = 1e-3
     tau = 0.03
     num, den = [1], [tau ** 2, 2 * tau, 1]
 
-    t, x, yhat = run_synapse(Simulator, seed, Alpha(tau), dt=dt)
-    y = LinearFilter(num, den).filt(x, dt=dt, y0=0)
+    synapse = Alpha(tau)
+    t, x, yhat = run_synapse(Simulator, seed, synapse, dt=dt)
+    y = LinearFilter((num, den)).filt(x, dt=dt)
+
+    assert signals_allclose(t, y, yhat, delay=dt, atol=5e-6, plt=plt, allclose=allclose)
+
+    gains = np.abs(synapse.evaluate([0, synapse.cutoff, 1e16]))
+    assert allclose(gains, [1, 1 / np.sqrt(2), 0])
+
+
+def test_double_exp(Simulator, plt, seed, allclose):
+    dt = 1e-3
+    synapse = DoubleExp(0.01, 0.002)
+    t, x, yhat = run_synapse(Simulator, seed, synapse, dt=dt)
+    y = synapse.filt(x, dt=dt)
+    assert signals_allclose(t, y, yhat, delay=dt, atol=5e-6, plt=plt, allclose=allclose)
+
+    check_synapse = Lowpass(0.002).combine(Lowpass(0.01))
+    assert np.allclose(check_synapse.tf[0], synapse.tf[0])
+    assert np.allclose(check_synapse.tf[1], synapse.tf[1])
```

Comment:
Added this test. Was expecting there to be a `.num` and `.den` attribute when I first tried this.